docs/release-notes/release-notes-0.19.0.md

```diff
@@ -224,6 +224,16 @@ The underlying functionality between those two options remain the same.
 
 
 ## Breaking Changes
+
+* **Warning:** The following RPCs will be removed in the next release (0.20):
```

Comment:
Looks good, thank you! The only question is if we want to give it more time and schedule the removal for 0.21, also since technically, we only deprecate `SendPaymentSync` and `SendToRouteSync` only with this PR @saubyk. It would be good to know how much usage those endpoints still see in popular projects.

Reply:
Moreover I think we should also highlight the corresponding REST methods here